app/src/main/res/values/colors.xml

```diff
@@ -26,4 +26,5 @@
     <color name="accent">#F57C00</color>
     <color name="error_color">#e10a0a</color>
     <color name="textPrimary">#FFFFFF</color>
+    <color name="dialogButtonText">#e10a0a</color>
```

Comment:
I think, it would be better to set here value to `@colors/error_color` (I mean set here value to the link to `error_color` value